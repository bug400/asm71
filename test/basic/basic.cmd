for %%I in (*.asm) do (
   if exist %%~nI.lex del %%~nI.lex
   if exist %%~nI.lst del %%~nI.lst
   asm71 %%~nI.asm,%%~nI.lex,%%~nI.lst,R2
   echo Comparing lst file
   diff %%~nI.lst reference\%%~nI.lst
   echo Comparing lex file
   diff %%~nI.lex reference\%%~nI.lex
   if exist %%~nI.lex del %%~nI.lex
   if exist %%~nI.lst del %%~nI.lst
)
