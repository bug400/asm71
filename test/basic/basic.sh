for i in *.asm ; do
    f=`basename ${i} .asm`
    asm71 ${i},${f}.lex,${f}.lst,R2
    echo "Comparing lst file"
    diff ${f}.lst reference/${f}.lst
    echo "Comparing lex file"
    diff ${f}.lex reference/${f}.lex
    rm -f ${f}.lex
    rm -f ${f}.lst
done
