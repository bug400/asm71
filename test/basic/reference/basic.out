
ASM71 
 


 Source : bsp.asm
 Object : bsp.lex
Listing : bsp.lst

Source Lines :     55           Errors :     0
Comparing lst file
Comparing lex file

ASM71 
 


 Source : memt.asm
 Object : memt.lex
Listing : memt.lst

Source Lines :    660           Errors :     0
Comparing lst file
Comparing lex file
